calculator-app/src/main/java/camp/nextstep/edu/calculator/CalculatorViewModel.kt

```diff
@@ -0,0 +1,43 @@
+package camp.nextstep.edu.calculator
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import camp.nextstep.edu.calculator.domain.Calculator
+import camp.nextstep.edu.calculator.domain.Expression
+import camp.nextstep.edu.calculator.domain.Operator
+import kotlin.coroutines.coroutineContext
+
+class CalculatorViewModel(initFormula: List<Any> = emptyList()) : ViewModel() {
+    private val calculator = Calculator()
+    private var expression = Expression(initFormula)
+
+    private var _formula = MutableLiveData(expression)
+    val formula: LiveData<Expression>
+        get() = _formula
```

Comment:
initFormula값을 바로 _formula 초기화에 사용해도 되지 않을까요? 지금은 작은 ViewModel이지만 나중에 앱이 커질수록 클래스 내 프로퍼티가 많이 생기게 되어, 이런 값들은 꼭 선언하지 않는 것이 관리하기 편할 수 있어요.
```suggestion
class CalculatorViewModel(initFormula: List<Any> = emptyList()) : ViewModel() {
    private val calculator = Calculator()

    private var _formula = MutableLiveData(Expression(initFormula))
    val formula: LiveData<Expression>
        get() = _formula
```